physics/physpkg.F90

```diff
@@ -1948,6 +1997,54 @@ subroutine tphysac (ztodt,   cam_in,  &
                   fh2o, surfric, obklen, flx_heat, cmfmc, dlf, det_s, det_ice, net_flx)
        end if
 
+#ifdef AEROCOM
+       !  Estimating hygroscopic growth by use of linear interpolation w.r.t. mass 
+       !  fractions of each internally mixed component for each mode (kcomp).
+       !
```

Comment:
I realize `AEROCOM` is not expected to work in this PR but if you want to include the code below, can you at least define the variables it uses (also inside of `#ifdef AEROOM`)?

Reply:
Good point. I've actually removed the AEROCOM stuff from physpkg - since it really looks like it was not doing anything. We can go back and revisit this when we actually turn on AEROCOM.